README.md

```diff
@@ -2,7 +2,7 @@
 
 A fully accessible and flexible React modal built according [WAI-ARIA Authoring Practices](http://www.w3.org/TR/wai-aria-practices/#dialog_modal).
 
-This module provides a "smart" minimally styled component to wrap you "dumb" fully styled component. It provides the following features, while giving you complete control of the content:
+This module provides a minimally styled "Container" component to wrap your fully-styled "Presentational" component. It provides the following features, while giving you complete control of the content:
```

Comment:
👍 👍 

Could we lowercase those words in quotation marks?

While we're at it, maybe we should provide a link to the article you mentioned, in case a reader doesn't know what these terms mean?